actors/builtin/miner/policy.go

```diff
@@ -146,19 +146,6 @@ func CanPreCommitSealProof(s abi.RegisteredSealProof, nv network.Version) bool {
 	return ok
 }
 
-// List of proof types for which sector lifetime may be extended.
```

Comment:
These will need to stay to support the version 10 code path.  (See comment above)

Reply:
These names can change to indicate they are only in scope for network version 10 and lower.  i.e. `ExtensibleProofTypesBeforeNV11`, `CanExtendSealProofTypeBeforeNV11`